go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/dynatrace-oss/dynatrace-monitoring-as-code
 
 require (
 	github.com/golang/mock v1.5.0
-	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76 // indirect
+	github.com/google/addlicense v0.0.0-20210428195630-6d92264d7170 // indirect
```

Comment:
Why did this change?

Reply:
When running the make target for adding/checking licenses it will run go get to ensure that 
the tool is installed. As a side-effect the addlicense version is updated in the mod file.